### PR TITLE
Cleanup

### PR DESCRIPTION
**Pull request by zippy**
this is a clone of @willemolding branch, plus a docker fix for the needed QTversion. Checking to see if CI builds...

@lucksus:
This makes the CI pass again after several changes during DevCamp #1 sprint
* Update to json string
* qt in docker
* replace hcshell with holochain-nodejs
* remove hcshell build from docker

### CODE REVIEW

File: zomes/blog/code/src/blog.rs [hunk 1]
```diff
@@ -1,135 +1,138 @@
...
 
-pub fn handle_check_sum(num1: u32, num2: u32) -> serde_json::Value {
-    #[derive(Serialize)]
+#[derive(Serialize, Deserialize, Debug, DefaultJson)]
+struct AddressRespose {
+    address: HashString
+}
+
+#[derive(Serialize, Deserialize, Debug, DefaultJson)]
+struct MultiAddressRespose {
```
Comment (Connoropolous) [inline zomes/blog/code/src/blog.rs]: "Respose" -> "Response"

Comment (zippy) [reply zomes/blog/code/src/blog.rs]: fixed

File: test/package.json [hunk 2]
```diff
@@ -10,10 +10,12 @@
   },
   "dependencies": {
     "tape": "^4.9.1",
-    "json3": "*"
+    "json3": "*",
+    "@holochain/holochain-nodejs": "*"
```
Comment (Connoropolous) [inline test/package.json]: I'd highly recommend against using `*` for nodejs deps like this

File: test/package.json [hunk 3]
```diff
@@ -10,10 +10,12 @@
...
   },
   "scripts": {
     "build": "webpack",
-    "watch": "webpack --watch"
+    "watch": "webpack --watch",
```
Comment (Connoropolous) [inline test/package.json]: you could take out `watch` -> `webpack` command

**Review by willemolding — APPROVED** [review]

**Review by Connoropolous — APPROVED** [review]

**Pull request merged**